sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/image_limit_settings.py

```diff
@@ -62,6 +62,13 @@ class ImageLimitSettings(RestTranslatableMixin):
         )
         # Submit the AutoML job.
         image_instance_segmentation_job.submit()
+
```

Comment:
There seems to be an inline example right above this, perhaps move it to the samples file so that it is checked by CI

Reply:
fixed